README.rst

```diff
@@ -0,0 +1,46 @@
+Apache Arrow Cookbooks
+======================
```

Comment:
Some statements about pre-requisites (installing the various Arrow libraries) in terms of building/developing recipes might be helpful

Reply:
Should be done in 1c83515
In theory all dependencies required to build the cookbooks are installed by the makefile itself on both Python and R (in Python we provide a `requirements.txt`), if I missed anything else apart a working Python/R environment that is necessary for the cookbook to compile/run let me know.